client/verta/verta/operations/monitoring/summaries.py

```diff
@@ -267,11 +337,20 @@ def __init__(self, conn, conf, msg):
         self.name = msg.name
         self.type = msg.type_name  # TODO: hide me
 
+        alerts_query = SummaryQuery(ids=[self.id])
+        self._alerts = _entities.Alerts(
+            conn, conf, self.monitored_entity_id, base_summary_query=alerts_query
+        )
```

Comment:
How would you feel about moving these lines into `def alerts` since that's the only place these would be used anyway?